packages/analytics-v1.1/.size-limit.js

```diff
@@ -4,26 +4,20 @@
  */
 module.exports = [
   {
-    name: 'Core - CDN - v1.1',
-    path: 'dist/legacy/rudder-analytics.min.js',
+    name: 'Core (legacy build) - CDN - v1.1',
+    path: 'dist/cdn/legacy/rudder-analytics.min.js',
     gzip: true,
     limit: '36.93 kB',
   },
-  {
-    name: 'All Integrations - CDN - v1.1',
-    path: 'dist/legacy/js-integrations/*.min.js',
-    gzip: true,
-    limit: '415.3 kB',
-  },
   {
     name: 'Core - NPM - v1.1',
-    path: 'dist/npm-lib/index.js',
+    path: 'dist/npm/index.js',
     gzip: true,
     limit: '36.87 kB',
   },
   {
```

Comment:
Is this still needed as service worker is a separate package now?

Reply:
We have not started using the new decoupled service worker package yet. as soon as this happens will clean up the one included in v1.1